projects/element-ng/filter-bar/filter.ts

```diff
@@ -25,4 +25,10 @@ export interface Filter {
    * Status of filter pill
    */
   status: FilterStatusType;
+  /**
+   * Disables automatic translation of title and description text.
+   * When set to true, the title and description will be displayed as-is without
```

Comment:
Please see in the comments above, we normally use links to refer to other properties when we do proper comments 👍 

Also, please make sure the `title` and `description` fields are marked as `TranslatableString` types instead of plain `string`.

```suggestion
   * Disables automatic translation of {@link title} and {@link description} text.
   * When set to true, the {@link title} and {@link description} will be displayed as-is without
```